### DISCUSSION TITLE
Imaging Information Redout not uploaded connecting to NINA

### DISCUSSION THREAD

**Discussion by Maxastro59** [mworion/MountWizzard4#339] [Closed Feature requests] [answered]
Hello, I noticed that if MW4 is connected to the camera via NINA ASTAP plate solving fails. Browsing the MW windows I can see the Image Information window without the camera data that should be collected by NINA (image 1). Then I connected MW directly to the camera drivers and the information appeared in the window (img 2). So I think MW4 doesn't interface properly with NINA; in fact I should also wait to see the images of the plate solving fields but this is not the case. The same happens if in NINA the camera is connected to both the ASCOMN driver and the native one. It seems like the connection with NINA has something that doesn't work. I'll try MW4 again next night with this new camera setting.

![Img-1](https://github.com/mworion/MountWizzard4/assets/154596811/26d3a40b-b15a-4149-9d9f-2162b0b88d19)

![Img-2](https://github.com/mworion/MountWizzard4/assets/154596811/dd49c551-01ef-414f-978d-91879b7aa3ad)

**Comment by mworion** [comment, marked as answer]
Hi,
when MW4 connects directly to the camera driver (and you have set there the parameters of the Chip correctly), ASCOM provides an interface for reading these values and MW4 does so.
The NINA interface (which is actually a copy of the SGPro interface) does not support theses parameters, so you have to set them manually in MW4. 
Please take into account that it is also the same story for the telescope settings (focal length and aperture) with the telescope driver. 
Hope this helps.
Michel

**Comment by Maxastro59** [comment]
Many thanks

Massimo Bernardi
www.mbernardi.it

Inviato da iPhone
